myhpom/static/myhpom/page/_upload_requirements.scss

```diff
@@ -0,0 +1,55 @@
+/*
+    styles for /upload/requirements.html
+*/
+
+$invalid_color: #dc3545;
+
+.upload-requirements__div {
```

Comment:
This doesn't appear to be used anywhere; could be removed.

Reply:
ah, yes, I restyled to match what you had done. 

I'll be glad to fix that little testing bug – just wasn't sure if I should.